DOCPLAN-368: Restructuring docs for DITA migration (#112642)
openshift/openshift-docs: modules/about-using-gpu-operator.adoc

= Using the NVIDIA GPU Operator

[role="_abstract"]
You can use the NVIDIA GPU Operator to provision worker nodes for running GPU-accelerated virtual machines (VMs) in {VirtProductName}.

The NVIDIA GPU Operator manages NVIDIA GPU resources in an {product-title} cluster and automates tasks when preparing nodes for GPU workloads.
The NVIDIA GPU Operator can also facilitate provisioning complex artificial intelligence and machine learning (AI/ML) workloads.

[NOTE]
====

> The NVIDIA GPU Operator is supported only by NVIDIA. For more information, see "Obtaining Support from NVIDIA" in the Red{nbsp}Hat Knowledgebase.

Reviewer: ```suggestion
You can get support for the NVIDIA GPU Operator through NVIDIA. For more information, see "Obtaining Support from NVIDIA" in the Red{nbsp}Hat Knowledgebase.
```
